### PR TITLE
text review

### PR DESCRIPTION
**Pull request by Mavodeli**

### CODE REVIEW

File: Crabasite/Assets/Resources/english.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
+    "There seem to be large silicate deposits in these asteroids. That's exactly what I need to repair the windshield. Good thing I recently bought that engine-powered refinery module, I can use that to make the adhesive for the windshield. But first I have to find a way to break down the asteroids into smaller pieces that I can carry back to the ship.",
     "completedFindSilicate" :
     "This should be enough silicate. If I bring it to the engine I can use the refinery module to make the adhesive for the windshield.",
     "completedRefineSilicate" :
     "Got the adhesive. Time to head over to the windshield and seal the cracks.",
     "completedRepairWindshield" : 
-    "Ok, that should do it. Still can't believe how incredibly lucky I was to strand right next to those silicate deposits... Mental note: start saving up for one of these reinforced solid windshields with an environmental projection system.",
```
Comment (KarylReyne) [inline Crabasite/Assets/Resources/english.json:23]: the idea was not to refer to a kind of shield, but to something called ['environmental projection system'](https://cyberpunk.fandom.com/wiki/CrystalDome). We could add an explicit Reference to 'CrystalDome' to make the idea clearer

**Review by KarylReyne — CHANGES_REQUESTED** [review]

File: Crabasite/Assets/Resources/english.json (hunk 1 above)
Comment (KarylReyne) [reply Crabasite/Assets/Resources/english.json:23]: `"Ok, that should do it. Still can't believe how incredibly lucky I was to strand right next to those silicate deposits. Mental note: start saving up for one of these reinforced solid windshields with an environmental projection system. I think they're called CrystalDome or something..."`

Comment (Mavodeli) [reply Crabasite/Assets/Resources/english.json:23]: Saw this a bit too late, 
is 5371673566c68b0e925775644842f6bcde49bab9 good, or should we use the version you just sent?

**Review by KarylReyne — APPROVED** [review]

**Comment by Mavodeli** [comment]
We can use this PR for #63

**Review by KarylReyne — APPROVED** [review]

**Comment by Mavodeli** [comment]
(fixed conflict with new black hole comments), no changes to text. 
P.S. view the changes of this PR by clicking ["Files changed"](https://github.com/Mavodeli/spaceroguelite/pull/107/files)

**Review by KarylReyne — APPROVED** [review]

**Review by EmptyN8 — APPROVED** [review]

**Review by xeniawetzel — APPROVED** [review]

**Review by KarylReyne — APPROVED** [review]

**Pull request merged**